pages/common/vegeta.md

```diff
@@ -0,0 +1,24 @@
+# vegeta
+
+> Vegeta is both a command line utility and a library for HTTP load testing.
+> See also `ab`.
+
+- Launch an attack lasting 30 seconds:
+
+`echo "{{GET https://example.com}}" | vegeta attack -cert {{path/to/cert.pem}} -key {{path/to/key.pem}} -duration={{30s}}`
```

Comment:
I am trying to understand why would I need a cert and key to communicate with example.com ? I should already have the CA installed in my machine. So unless the server uses a self-signed cert, I shouldn't have to use any certs or keys.